src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2053,10 +2053,31 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
     if (admin)
         info[jss::load] = m_job_queue.getJson ();
 
+    auto const escalationMetrics = app_.getTxQ().getMetrics(
+        app_, *app_.openLedger().current());
     if (!human)
     {
         info[jss::load_base] = app_.getFeeTrack ().getLoadBase ();
         info[jss::load_factor] = app_.getFeeTrack ().getLoadFactor ();
+        if (escalationMetrics)
+        {
+            /* Json::Value doesn't support uint64, so clamp to max
+                uint32 value. This is mostly theoretical, since there
+                probably isn't enough extant XRP to drive the factor
+                that high.
+            */
+            constexpr std::uint64_t max =
```

Comment:
I weighed that option, but ruled it out because `load_base` and `load_factor` are numbers, and I didn't want to have to force consumers to deal with mixed types. Am I overestimating the issue?